src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,134 @@
+package ui;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    private static final String NAME = "Tartar";
```

Comment:
Noted on significant use of caps in final constants! Good job

Reply:
How nice of you.